tsl/src/nodes/vector_agg/plan.c

```diff
@@ -500,11 +500,17 @@ vectoragg_plan_possible(Plan *childplan, const List *rtable, VectorQualInfo *vqi
 	RangeTblEntry *rte = rt_fetch(customscan->scan.scanrelid, rtable);
 
 	if (ts_is_hypercore_am(ts_get_rel_am(rte->relid)))
+	{
 		vectoragg_plan_tam(childplan, rtable, vqi);
```

Comment:
Nit: The names of these functions could be a little better (e.g., starting with a verb). What they do is extract vector qual information into the memory pointed to by `vqi` but that is not clear from the name nor from any comments associated with the function, nor here.

Reply:
I'll leave function name refactoring out of this fix.